src/components/structures/FilePanel.tsx

```diff
@@ -173,7 +181,7 @@ class FilePanel extends React.Component<IProps, IState> {
         // the event index to fulfill the pagination request. Asking the server
         // to paginate won't ever work since the server can't correctly filter
         // out events containing URLs
-        if (room && client.isRoomEncrypted(roomId) && eventIndex !== null) {
+        if (room && this.state.isRoomEncrypted && eventIndex !== null) {
             return eventIndex.paginateTimelineWindow(room, timelineWindow, direction, limit);
         } else {
             return timelineWindow.paginate(direction, limit);
```

Comment:
Will anything go wrong if this branch gets executed while `this.state.isRoomEncrypted` is null?

Reply:
I conditioned the TimelinePanel under the isRoomEncryptedLoaded check. So this case should not happen now